Makefile

```diff
@@ -5,6 +5,7 @@ help:	## Show all Makefile targets.
 
 format:	## Run code autoformatters (black).
 	black .
+	black docs/examples/* --preview -l 79
```

Comment:
Will there be `black` churn with alternating line lengths used here?  Also, imo it's better to use `--line-length` over `-l` for readability in `Makefile`s

Reply:
👍 on using `line-length` for readability. We're actually running into some issues in our linting step as well here, as we run `black-jupyter` in our lint pre-commit hooks that aren't configured with this 79 line-length for our docs/*.ipynb

Reply:
Yeah I know what you mean.  The solution here is:

1. Make one `pre-commit` hook with a `name`: `black-docs`
    - Use `files` to filter for `docs/`
2. Make another `pre-commit` hook with a `name`: `black-src`
    - Use `exclude` to filter out `docs/`

Then in `Makefile`'s `format`:

```makefile
format:	## Run code autoformatters (black).
    pre-commit install
	pre-commit run black-src
	pre-commit run black-docs
```

Reply:
This is great. Thanks.